config/api/src/main/java/org/jboss/windup/config/metadata/AbstractRulesetMetadata.java

```diff
@@ -33,6 +33,12 @@ public String getID()
     }
 
     @Override
+    public String getDescription()
+    {
+        return null;
```

Comment:
If this is intended to be a default, it should be "" empty string but not null IMO.